pkg/process/checks/pod.go

```diff
@@ -59,6 +59,8 @@ func (c *PodCheck) IsEnabled() bool {
 		return false
 	}
 
+	log.Warn("This check will be deprecated in 7.51.0 and move to corechecks")
```

Comment:
The concept of core checks feels like an implementation detail that users are probably not aware of

```suggestion
	log.Warn("This Process Agent check will be deprecated in 7.51.0 and moved to the Node Agent")
```

Reply:
[78fa929](https://github.com/DataDog/datadog-agent/pull/19478/commits/78fa92963f08ddbb3e6049590de1b8a0cf0a910e)